include/quda_cuda_api.h

```diff
@@ -1,68 +1,136 @@
 #pragma once
 
 #ifndef __CUDACC_RTC__
-#include <cuda.h>
-#include <cuda_runtime.h>
+#include <quda_backend_api.h>
```

Comment:
This file looks mostly fine, but there are many comments that mistakenly say that the wrapper is to a quda* function, where it should be cuda* function of course.

Reply:
All taken care of.